bigbluebutton-html5/imports/api/2.0/users/server/modifiers/userEjected.js

```diff
@@ -23,6 +25,18 @@ export default function userEjected(meetingId, userId) {
     },
   };
 
+  const user = VoiceUsers.findOne({ meetingId, voiceUserId: userId });
+
+  if (user) {
```

Comment:
Correct me if I'm wrong, I thought we were supposed to:
1. Fetch a userVoice object from VoiceUsers in the [kickUser.js](https://github.com/KDSBrowne/bigbluebutton/blob/3b6aed34ae46420d7707ad7772d63d9eba9184a6/bigbluebutton-html5/imports/api/2.0/users/server/methods/kickUser.js#L22)
2. Check if the user is voice only
3. If 2 is true - send an `EjectUserFromVoiceCmdMsg` event instead of `UserEjectedFromMeetingEvtMsg`
4. return

This doesn't look like the right place or the right action to me.
If the user is in audio when somebody kicked him out of the meeting - we receive a separate event to remove it from VoiceUsers - `UserLeftVoiceConfToClientEvtMsg`, theoretically there is no need to clean the collection here.